LBH.AdultSocialCare.Api/V1/Profiles/MappingProfile.cs

```diff
@@ -24,7 +27,6 @@ public MappingProfile()
             CreateMap<DayCarePackageForCreationDomain, DayCarePackage>();
             CreateMap<DayCarePackageForUpdateDomain, DayCarePackage>();
             CreateMap<DayCarePackage, DayCarePackageDomain>()
-                .ForMember(dc => dc.PackageName, opt => opt.MapFrom(b => b.Package.PackageName))
```

Comment:
Is there no package name anymore? If there is update to new reference. Package name will always be null in this case because there is no mapping to it

Reply:
With @petemade changes I think we don't need package name in each package. We already know which package they are ?

Reply:
In that case, let the mapping return `"Day Care package"` like this `.ForMember(dc => dc.PackageName, opt => opt.MapFrom(b => "Day Care package"))`

Reply:
changed.